.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+/ @maxence-charriere
```

Comment:
🤔

Reply:
wanna be in @marktran ?

Reply:
Ya can you add @marktran  and @maxchehab